packages/dx-react-chart-bootstrap4/src/templates/grid/line.test.jsx

```diff
@@ -0,0 +1,35 @@
+import * as React from 'react';
+import { shallow } from 'enzyme';
+import { Line } from './line';
+
+const defaultProps = {
+  x1: 1, y1: 2, x2: 3, y2: 4,
+};
+
+describe('Line', () => {
+  it('should render line', () => {
+    const {
+      x1, y1, x2, y2,
+    } = shallow((
+      <Line {...defaultProps} />
+    )).find('line').props();
+
+    expect(x1).toBe(1);
+    expect(y1).toBe(2);
+    expect(x2).toBe(3);
+    expect(y2).toBe(4);
+  });
+
+  it('should pass the className prop to the root element', () => {
+    const tree = shallow(<Line {...defaultProps} className="custom-class" />);
+
+    expect(tree.is('.dx-c-bs4-stroke-current-color')).toBeTruthy();
+    expect(tree.is('.custom-class')).toBeTruthy();
```

Comment:
Use more simple way... expect once
`expect(tree.is('.dx-c-bs4-stroke-current-color.custom-class')).toBeTruthy();`